src/stylesheets/_colors.scss

```diff
@@ -3,3 +3,5 @@ $medium-gray: #d9d9d9;
 $turquoise: #009BC3;
 $easi-bright-blue: #0071bc;
 $easi-blue: #005EA2;
+$easi-dark-blue: #1a4480;
```

Comment:
https://designsystem.digital.gov/design-tokens/color/theme-tokens/
'primary-dark' is available for this